helm/block-producer/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: block-producer
 description: A Helm chart for Mina Protocol's block producing network nodes
 type: application
-version: 0.4.6
+version: 0.5.0
 appVersion: 1.16.0
 annotations:
   artifacthub.io/changes: |
```

Comment:
hey, we need to update these changes and run `make changelog` to go along with the update.

Reply:
:+1: ran make changelog with new annotation